### PR TITLE
fix: set correct musig status icons

### PR DESCRIPTION
**Pull request by w3ea**
## Summary

https://app.clickup.com/t/12w7gfu

## Checklist

-   [x] My changes look good in both light AND dark mode
-   [ ] The change is not hardcoded to a single network, but has multi-asset in mind
-   [ ] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

**Comment by dated** [comment]
The icons used will appear with a different stroke width, since one is being scaled down while the other is not, this may need an additional clock icon, or you could use the regular checkmark icon instead of the small one.

**Comment by w3ea** [comment]
yeah i think we need a new clock icon (needs another PR)
the regular checkmark icon doesn't match the design, and it's very thin

image (1)

**Comment by dated** [comment]
Well, that second PR could have the same name as this one, so it might as well be fixed here 😂

**Comment by w3ea** [comment]
then i'll change the task, because it was just about the checkmark badge :oleglala:

File: src/app/assets/svg/clock-small.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="clock" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 10 10" xml:space="preserve"><g transform="translate(1.5 1.5)" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10"><circle cx="3.5" cy="3.5" r="4"/><path d="M3.5 3.5V2m.1 1.6l.9.9"/></g></svg>
```
Comment (dated) [inline src/app/assets/svg/clock-small.svg:1]:
```suggestion
<svg version="1.1" id="clock-small" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 10 10" xml:space="preserve"><g transform="translate(1.5 1.5)" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10"><circle cx="3.5" cy="3.5" r="4"/><path d="M3.5 3.5V2m.1 1.6l.9.9"/></g></svg>
```

**Review by samharperpittam — APPROVED** [review]

**Pull request merged**